src/components/color-picker/color-picker.tsx

```diff
@@ -0,0 +1,97 @@
+import { Component, h, Prop, State } from '@stencil/core';
+
+/**
+ * This component enables you to select a swatch from out color palette, simply
+ * by clicking on it. You can then copy the css variable name of the chosen color
+ * and use it where desired.
+ *
+ * The color picker can also show you a preview of any valid color name or color value.
+ *
+ * @exampleComponent limel-example-color-picker
+ */
+
+@Component({
+    tag: 'limel-color-picker',
+    shadow: true,
+    styleUrl: 'color-picker.scss',
+})
+export class ColorPicker {
+    /**
+     * Name or code of the chosen color
+     */
+    @Prop({ reflect: true })
+    public color: string;
+
+    /**
+     * The label of the input field.
+     */
+    @Prop({ reflect: true })
+    public label: string;
+
+    /**
+     * The helper text of the input field.
+     */
+    @Prop({ reflect: true })
+    public helperText: string;
+
+    /**
+     * The title text of the input field.
+     */
+    @Prop({ reflect: true })
+    public tooltipLabel: string;
+
```

Comment:
We need to emit a `change` event from this component, otherwise the consumer will never know when the value has changed